charmhelpers/core/hookenv.py

```diff
@@ -57,6 +58,14 @@
                  'This may not be compatible with software you are '
                  'running in your shell.')
 
+
+class WL_STATES(Enum):
```

Comment:
What does `WL_` stand for?

Reply:
Assuming "workload", why is it not expanded fully?

Reply:
I'll address this in a subsequent PR